packages/sdks/javascript/Makefile

```diff
@@ -0,0 +1,7 @@
+test:
```

Comment:
Please sort the targets alphabetically inside a `Makefile`, for increased readability.

Reply:
@jfoliveira resolved